README.md

```diff
@@ -73,7 +72,7 @@ brew install tlrc
 Then you have direct access to simplified, easy-to-read help for commands, such as `tar`,
 accessible through typing `tldr tar` instead of the standard `man tar`.
 
-If you want an offline version without installing any software,
+If you don't want to install any software,
```

Comment:
Contractions are used a lot in the README. Do we really want to change that everywhere?

> Maybe ***you're*** new to the command-line world? Perhaps ***you're*** just a little rusty or can't always recall the arguments for commands like `lsof`, or `tar`?

> If ***you'd*** like to contribute to translations

> If ***it's*** your first time contributing

Reply:
IMO it's better to use "do not" over "don't" only when a more rigid tone is necessary

Reply:
I asked 4 different AI chatbots (ChatGPT, Gemini, Claude and Perplexity) regarding the _tonality_ of the two sentences ("do not" and "don't"). All 4 AI chatbots said "do not" is more formal and authoritative while "don't" is more casual/friendly. 

IMO "don't" is fine here.

Reply:
As a native english speaker, I suggest that contractions are fine and shouldn't be removed.

If this was for example a formal academic document then this would be a different story, but I don't think this README is formal enough to count there.

This is an open source project: I suggest `don't` rather than `do not` is more friendly and inviting, which is an important quality of our README -- something that is often the first thing new people / contributors / etc see :-)